### PR TITLE
[RHINENG-11403] Create a new Makefile entry to run the inv_export_service.py

### PR DESCRIPTION
**Pull request by strider**
This PR adds a new Makefile entry to run the `inv_export_service.py` script.

# Overview

This PR is being created to address [RHINENG-11403](https://issues.redhat.com/browse/RHINENG-11403).

## PR Checklist

- [ ] Keep PR title short, ideally under 72 characters
- [ ] Descriptive comments provided in complex code blocks
- [ ] Include raw query examples in the PR description, if adding/modifying SQL query
- [ ] Tests: validate optimal/expected output
- [ ] Tests: validate exceptions and failure scenarios
- [ ] Tests: edge cases
- [ ] Recovers or fails gracefully during potential resource outages (e.g. DB, Kafka)
- [ ] Uses [type hinting](https://docs.python.org/3/library/typing.html), if convenient
- [ ] Documentation, if this PR changes the way other services interact with host inventory
- [ ] Links to related PRs

## Secure Coding Practices Documentation Reference

You can find documentation on this checklist [here](https://github.com/RedHatInsights/secure-coding-checklist).

## Secure Coding Checklist

- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

**Comment by roliveri** [comment]
This doesn't require any environment variables for configuration?

**Comment by strider** [comment]
> This doesn't require any environment variables for configuration?

oh, you're right ... I have everything in my `.env` file ...

File: Makefile [hunk 1]
```diff
@@ -26,6 +26,9 @@ run_inv_web_service:
 run_inv_mq_service:
 	KAFKA_EVENT_TOPIC=platform.inventory.events PAYLOAD_TRACKER_SERVICE_NAME=inventory-mq-service INVENTORY_LOG_LEVEL=DEBUG BYPASS_TENANT_TRANSLATION=true python3 inv_mq_service.py
 
+run_inv_export_service:
+	python3 inv_export_service.py
```
[the remark this answers is not in the document]
Comment (strider) [reply Makefile:30]: Yes, thanks. Rich already spotted it above! 👍

**Review by jpramos123 — APPROVED** [review]

**Comment by strider** [comment]
@roliveri @thearifismail @jpramos123 

I've added them and updated the `README.md` file as well.

**Review by kruai — APPROVED** [review]

**Review by thearifismail — APPROVED** [review]

**Review by roliveri — APPROVED** [review]

**Review by jpramos123 — APPROVED** [review]

**Pull request merged**